app/controllers/ops_controller.rb

```diff
@@ -321,8 +316,6 @@ def features
       }
     ]
 
-    features.push(:role => "ops_db", :name => :vmdb, :title => _("Database")) if ::Settings.ui.display_ops_database
```

Comment:
@jrafanie FYI, if you didn't see there is a Setting for this that can be dropped (and removed in a data migration)